guides/common/modules/proc_installing-the-discovery-service.adoc

```diff
@@ -1,18 +1,50 @@
 [id="Installing_the_Discovery_Service_{context}"]
 = Installing the Discovery Service
 
-Complete the following procedure to enable the Discovery service on {SmartProxyServer}.
+On the applicable operating system, complete the following procedure to enable the Discovery service on {SmartProxyServer}.
 
-.Procedure
+* xref:#installing-discovery-el-8[{EL} 8]
+* xref:#installing-discovery-el-7[{EL} 7]
```

Comment:
There is no difference in instructions between EL 7 & 8: the installer takes care of all of that. Please align on only using the installer for both EL 7 & 8.

Reply:
Addressed :+1: